agentflow/server.py

```
@@ -0,0 +1,1918 @@
+    @app.post("/upload/directory")
+    async def upload_directory(
+        directory_path: str = Form(...),
+        client_name: str = Form(""),
+        matter_type: str = Form(""),
+    ):
+        """Upload all files from a directory, auto-detect client name and matter type."""
+        if not os.path.isdir(directory_path):
+            raise HTTPException(status_code=400, detail=f"Directory not found: {directory_path}")
+
+        all_files = []
+        for root, dirs, files in os.walk(directory_path):
```

Ruff (0.15.7)

[warning] Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)